docs/indepth/installation.rst

```diff
@@ -215,7 +215,7 @@ Using Amazon Web Services
 
```

Comment:
Not part of your patch, but the step above is missing a `@` after `root`.

Also, previous to that, the code block starting with "You should..." is superfluous, repeating the subsequent step (and using a different example hostname to boot).